Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.8-slim-buster
+
+RUN python -m pip install --upgrade pip
+
+WORKDIR  /code
+
+
+COPY ./requirements.txt .
```

Нет смысла два раза копировать этот файл, сразу копируй всё содержимое
```suggestion
COPY . .
```

---

хотел вынести в отдельный слой, чтобы не устанавливались зависимости каждый раз

---

Точно, слышал об этом уже однажды. Добавь, пожалуйста, комент, почему отдельно копируется файл зависимостей.